TRAMS-API/Models/Downstream/D365/SearchProjectsD365Model.cs

```diff
@@ -0,0 +1,36 @@
+﻿using API.Models.D365.Enums;
+using Newtonsoft.Json;
+using System;
+
+namespace API.Models.Downstream.D365
+{
+    public class SearchProjectsD365Model
+    {
+        [JsonProperty("sip_academytransfersprojectid")]
+        public Guid ProjectId { get; set; }
+
+        [JsonProperty("sip_projectname")]
+        public string ProjectName { get; set; }
+
+        [JsonProperty("sip_projectinitiatorfullname")]
+        public string ProjectInitiatorFullName { get; set; }
+
+        [JsonProperty("sip_projectinitiatoruniqueid")]
+        public string ProjectInitiatorUid { get; set; }
+
+        [JsonProperty("sip_projectstatus")]
+        public ProjectStatusEnum ProjectStatus { get; set; }
+
+        public override bool Equals(object obj)
+        {
+            return ProjectId == ((SearchProjectsD365Model)obj).ProjectId;
```

Comment:
Suggestion:  Unlikely to be the case, but to cover all bases I'd potentially use a try cast rather than a direct cast here in case obj is something other than what you expect.  (obj as SearchProjectD365Model)?.ProjectId

Reply:
Done